app/models/concerns/blacklight/document/export.rb

```diff
@@ -40,56 +40,56 @@
 #
 module Blacklight::Document::Export
 
-  ##  
+  ##
   # Register exportable formats supported by the individual document.
   # Usually called by an extension in it's self.extended method, to
-  # register the formats that extension can export. 
-  # 
+  # register the formats that extension can export.
+  #
   # some_document.will_export_as(:some_format, "application/type") means
   # that the document (usually via an extension) has a method
   # "export_as_some_format" which returns a String of content that
-  # is described by the mime content_type given. 
-  # 
+  # is described by the mime content_type given.
+  #
   # The format name should ideally _already_ be registered with
   # Rails Mime::Type, in your application initializer, representing
   # the content type given.  However, this method will attempt to
   # register it using Mime::Type.register_alias if it's not previously
-  # registered. This is a bit sketchy though. 
+  # registered. This is a bit sketchy though.
   def will_export_as(short_name, content_type = nil)
     #Lookup in Rails Mime::Type, register if needed, otherwise take
     # content-type from registration if needed.
     if defined?(Mime) && Mime[short_name.to_sym]
       content_type ||= Mime[short_name.to_sym]
     else
       # not registered, we need to register. Use register_alias to be least
-      # likely to interfere with host app. 
+      # likely to interfere with host app.
       Mime::Type.register_alias(content_type, short_name)
     end
 
     export_formats[short_name] =  { content_type: content_type }
   end
-  
+
   # Collects formats that this doc can export as.
   # Returns a hash, keys are format short-names that can
   # be exported. Hash includes:
-  #  :content-type => mime-content-type
+  #  :content-type => { content_type: content_type }
```

Comment:
I think this was intended to be a placeholder for a string, so something like this would be more correct:
```ruby
:content_type => 'sample/content-type'
```